app/server/express.js

```diff
@@ -68,9 +65,16 @@ const swaggerOptions = {
 
 const specs = swaggerJsDoc(swaggerOptions);
 
+const uiOptions = {
+  customSiteTitle: "HWP Swagger Docs",
+  customCss: `
```

Comment:
Hold on. I can customize the look of the swagger docs?! This is a very nice feature. Good find.

Reply:
Lol like I dmed you, this was hard to find. Swagger documentation is either confusing, incomplete, or non existent. Definitely a nice piece of knowledge we can use in future projects.

Reply:
My plan if we were to do any more extensive css was to put it into its own css file. It might mean we put a require('swagger-ui.css') as the value for customCss